mmv1/products/vertexai/terraform.yaml

```
@@ -13,6 +13,29 @@
 
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
+  Tensorboard: !ruby/object:Overrides::Terraform::ResourceOverride
+    skip_sweeper: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "vertex_ai_tensorboard"
+        skip_import_test: true
+        primary_resource_id: "tensorboard"
+        vars:
+          display_name: "terraform"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "vertex_ai_tensorboard_full"
+        skip_import_test: true
```

Why do these tests use `skip_import_test`?

---

It's because we need to use `ImportStateVerifyIgnore: []string{"region", "project"}` instead of `ImportStateVerifyIgnore: []string{"region"}`. There is a difference in the `name` field (`projects/{project}/locations/{location}/tensorboards/{tensorboard}`) of imported resources where instead of project_id we are getting project_number. Hence, I've used `skip_import_test`

---

I think that should be fine, since `name` is an output-only field.

You can use `ignore_read_extra` here & above instead of `skip_import_test` to just ignore the one additional field:

```suggestion
        ignore_read_extra:
          - project
```

If the API returns project numbers - will users get a permadiff if they try to set the `project` field using a project id?

---

No, User won't face state drift if the `project` field is set to project id. It's just that import is forming the `name` field using the `replaceVars` function which takes the project_id and this gets changed when the resource is successfully imported.

---

I've updated examples to include `ignore_read_extra` for the tests.